pkg/client/localsubjectaccessreview.go

```diff
@@ -63,7 +63,7 @@ func (c *localSubjectAccessReviews) Create(sar *authorizationapi.LocalSubjectAcc
 		}
 		deprecatedResponse := &authorizationapi.SubjectAccessReviewResponse{}
 
-		req, err := overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
+		req, err = overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
```

Comment:
Agreed, let's use a different name.

On Tue, May 31, 2016 at 3:34 PM, David Eads <notifications@github.com>
wrote:

> In pkg/client/localsubjectaccessreview.go
> <https://github.com/openshift/origin/pull/9093#discussion_r65181512>:
>
> > @@ -63,7 +63,7 @@ func (c *localSubjectAccessReviews) Create(sar *authorizationapi.LocalSubjectAcc
> >  		}
> >  		deprecatedResponse := &authorizationapi.SubjectAccessReviewResponse{}
> >
> > -		req, err := overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
> > +		req, err = overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
>
> So I thought this created a new req variable in this scope instead of
> re-using the one from above. I don't like re-using variables. If we want to
> remove the marker, you could change it to a different variable name?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9093/files/02f0b0e3dd2a22ca63331451be236c6c6c539959#r65181512>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADuFf-g3C6zDAxoF3eRRUcuqhftvVIArks5qHDj0gaJpZM4IqgNW>
> .
>